src/components/users/user-signup/ko/runtime/user-signup.html

```diff
@@ -46,15 +46,6 @@
             </div>
             <!-- /ko -->
 
-            <!-- ko ifnot: isConsentRequired -->
-            <div class="row">
-                <p>
-                    Please review Terms of Use.
-                    <a href="#" class="btn-link" data-bind="click: toggleRegistrationTerms, text: showHideLabel"></a>
-                </p>
-            </div>
-            <!-- /ko -->
-
```

Comment:
why did you remove this? Customers can set terms of use content but do not set that consent required for it.